bundles/org.openhab.binding.modbus.studer/README.md

```diff
@@ -0,0 +1,142 @@
+
+#### BSP 
+
+All channels read for a BSP device
+
+| Channel            | Type                     | Description           |
+| ------------------ | ------------------------ | --------------------- |
+| power              | Number:Power             | Power                 |
+| batteryVoltage     | Number:ElectricPotential | Battery voltage       |
+| batteryCurrent     | Number:ElectricCurrent   | Battery current       |
+| stateOfCharge      | Number:Dimensionless     | State of Charge       |
+| batteryTemperature | Number:Temperature       | Battery temperature   |
+
+#### Xtender 
+
+All channels read for a Xtender device
+
+| Channel           | Type                     | Description             |
+| ----------------- | ------------------------ | ----------------------- |
+| inputVoltage      | Number:ElectricPotential | Input voltage           |
+| inputCurrent      | Number:ElectricCurrent   | Input current           |
+| inputActivePower  | Number:Power             | Input active power      |
+| inputFrequency    | Number:Frequency         | Input frequency         |
+| outputVoltage     | Number:ElectricPotential | Output voltage          |
+| outputCurrent     | Number:ElectricCurrent   | Output current          |
+| outputActivePower | Number:Power             | Output active power     |
+| outputFrequency   | Number:Frequency         | Output frequency        |
+| operatingState    | String                   | Operating state         |
+| stateInverter     | String                   | State of the inverter   |
+
+#### VarioTrack 
+
+All channels read for a VarioTrack device
+
+| Channel              | Type                     | Description                               |
+| -------------------- | ------------------------ | ----------------------------------------- |
+| modelVarioTrack      | String                   | Model of VarioTrack                       |
+| voltagePVGenerator   | Number:ElectricPotential | Voltage of the PV generator               |
+| powerPVGenerator     | Number:Power             | Power of the PV generator                 |
+| productionCurrentDay | Number:Energy            | Production in (kWh) for the current day   |
+| batteryVoltage       | Number:ElectricPotential | Battery voltage                           |
+| batteryCurrent       | Number:ElectricCurrent   | Battery current                           |
+| operatingMode        | String                   | Operating mode                            |
+| stateVarioTrack      | String                   | State of the VarioTrack                   |
+
+#### VarioString 
+
+All channels read for a VarioString device
+
+| Channel                 | Type                     | Description                                   |
+| ----------------------- | ------------------------ | --------------------------------------------- |
+| PVVoltage               | Number:ElectricPotential | PV voltage                                    |
```

Comment:
Any reason these channels have uppercase first character.

It also looks these channels could be designed as groups. So a group for PV, PV1 and PV2?

Reply:
PV I think is an acronym. I would like to group them into groups but I don't know how.